### DISCUSSION TITLE
https://greendolphindance.com/2024/01/19/%E8%A7%A3%E5%86%B3%E2%80%9C%E8%87%B4%E5%91%BD%E9%94%99%E8%AF%AF%EF%BC%9A%E6%97%A0%E6%B3%95%E8%AF%BB%E5%8F%96%E8%BF%9C%E7%A8%8B%E4%BB%93%E5%BA%93%E2%80%9D%E9%97%AE%E9%A2%98/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#14] [General]
# https://greendolphindance.com/2024/01/19/%E8%A7%A3%E5%86%B3%E2%80%9C%E8%87%B4%E5%91%BD%E9%94%99%E8%AF%AF%EF%BC%9A%E6%97%A0%E6%B3%95%E8%AF%BB%E5%8F%96%E8%BF%9C%E7%A8%8B%E4%BB%93%E5%BA%93%E2%80%9D%E9%97%AE%E9%A2%98/

Back to the rough ground!

https://greendolphindance.com/2024/01/19/%E8%A7%A3%E5%86%B3%E2%80%9C%E8%87%B4%E5%91%BD%E9%94%99%E8%AF%AF%EF%BC%9A%E6%97%A0%E6%B3%95%E8%AF%BB%E5%8F%96%E8%BF%9C%E7%A8%8B%E4%BB%93%E5%BA%93%E2%80%9D%E9%97%AE%E9%A2%98/

**Comment by ouuan** [comment]
我觉得最后两次错误单纯是连接不稳定；ssh-add 是后续不用再输入 ssh 密码（如果设了的话），KeepAlive 是连到交互式终端的时候防止长时间不操作断开连接，都和 git push 失败没啥关系（

**Reply by greendolphindance** [reply]
原来如此！

**Comment by ouuan** [comment]
另外更换梯子为什么没用呢，都封了吗（

**Reply by greendolphindance** [reply]
俺也不知道，确实clash里面换节点、把clash换成迷雾通等都无法解决

**Reply by greendolphindance** [reply]
但开其他网页等等是正常的！
